src/vm/wren_core.wren

```diff
@@ -323,6 +323,41 @@ class List is Sequence {
     return other
   }
 
+  sort() { sort {|a, b| a < b } }
```

Comment:
I suspect this might want to be `<=` for stable sort results, otherwise equal items will rearrange themselves on successive sorting, and won't be deterministic.

Reply:
Quick sort isn't stable anyway.

Reply:
https://stackoverflow.com/questions/13498213/quicksort-algorithm-stability

Reply:
It can be implemented in a manner that makes it stable, at the cost of memory consumption: https://www.geeksforgeeks.org/stable-quicksort/

As an alternative, we could use a merge sort, which has similar properties, plus stability. However, it would also have some memory consumption requirements.

Reply:
I know that, but unless we use a stable sort (which I suggested earlier), it's useless to discuss about `<` vs. `<=`.

Reply:
Is there a specific reason we want a stable sort? My idea for using a non-stable quick-sort was that it's highly memory efficient which is perfect for, in my opinion, virtual memory environments like the WrenVM. It might have a worst-case scenario of running at O(n^2) but on average it should run O(n logn).

If a more efficient algorithm is needed it can be left to the user.

Reply:
I think we can get away with it being unstable, so long as we document it clearly. From an embeddability point of view, the memory usage argument is quite compelling, in my opinion.

Reply:
@avivbeeri Thank you! and yes, I will have to redo the documentation